game.py

```diff
@@ -2,6 +2,7 @@
 import time
 from termcolor import colored, cprint # pip3 install termcolor
 from playsound import playsound       # pip3 install playsound
+from messages import random_rest_messages, random_mine_messages, random_fish_messages, random_combat_messages
```

Comment:
ah okay so that's how that works. I tried that with a lot of the global variables before i changed them to dictionaries. thanks